packages/payload/src/errors/ValidationError.ts

```diff
@@ -37,8 +39,35 @@ export class ValidationError extends APIError<{
         ? en.translations.error.followingFieldsInvalid_one
         : en.translations.error.followingFieldsInvalid_other
 
+    const req = results.req
+    delete results['req']
```

Comment:
I would add a short comment explaining the delete (to prevent req from being logged in the error cause, since we pass the results object to the error below), as it wasn't immediately clear to me when first reading the code